implementations/rust/Cargo.toml

```diff
@@ -25,6 +26,8 @@ default-members = [
     "router",
     "transport",
     "node",
+    "xeddsa",
+    "c/generate_bindings",
```

Comment:
This was out of default_members list for a purpose

Reply:
Except I didn't add c/generate_bindings, that was from a merge.